tests/CMakeLists.txt

```diff
@@ -54,14 +39,9 @@ foreach(label ${TESTSUITE_EXCLUDE})
     set(CTEST_EXTRA_ARGS ${CTEST_EXTRA_ARGS} -LE ${label})
 endforeach()
 
-add_custom_target(check COMMAND CTEST_OUTPUT_ON_FAILURE=1 ${CMAKE_CTEST_COMMAND} ${CTEST_EXTRA_ARGS}
+add_custom_target(check COMMAND CTEST_OUTPUT_ON_FAILURE=1 ${CMAKE_CTEST_COMMAND} 
```

Comment:
Why did it go away?

Reply:
I don't know, maybe after merge conflicts resolve